COPYING

```diff
@@ -20,6 +20,6 @@ Redistribution and use in source and binary forms, with or without modification,
 
     * Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
     * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
-    * Neither the name of Redis nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
+    * Neither the name of Valkey nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
```

Comment:
This is the original obligation from the BSD Redis license, so we aren't touching this.
```suggestion
    * Neither the name of Redis nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
```